apps/server/src/shared/domain/entity/boardnode/file-element-node.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { AnyBoardDo } from '../../domainobject';
+import { BoardNode, BoardNodeProps } from './boardnode.entity';
+import { BoardDoBuilder, BoardNodeType } from './types';
+
+@Entity({ discriminatorValue: BoardNodeType.FILE_ELEMENT })
+export class FileElementNode extends BoardNode {
```

Comment:
Why are all of Node Entities not having tests?

Reply:
Good question

Reply:
done!